include/mbedtls/ctr_drbg.h

```diff
@@ -45,7 +48,12 @@
 #define MBEDTLS_ERR_CTR_DRBG_FILE_IO_ERROR                -0x003A  /**< Read or write error in file. */
 
 #define MBEDTLS_CTR_DRBG_BLOCKSIZE          16 /**< The block size used by the cipher. */
+
+#if !defined(MBEDTLS_CTR_DRBG_USE_128_BIT_KEY) /* Use default 256-bit key for CTR_DRBG. */
```

Comment:
Please avoid “if not condition then x else y” unless there's a good reason. Change to `#if defined(MBEDTLS_CTR_DRBG_USE_128_BIT_KEY)` and swap the key sizes.

Reply:
fixed